src/os/ui/timelinepanel.js

```diff
@@ -233,3 +239,21 @@ os.ui.TimelinePanelCtrl.prototype.adjust = function() {
   os.MapContainer.getInstance().updateSize();
   os.dispatcher.dispatchEvent(os.MapEvent.GL_REPAINT);
 };
+
+
+/**
+ * Panel lock button click.
+ */
+os.ui.TimelinePanelCtrl.prototype.lock = function() {
```

Comment:
This function is called at `timelinepanel.html#26`, so it needs to have the `@export` tag added to the comment. This prevents the compiler from renaming the function and without it, it might not work in compiled mode.